genestack_client/file_filters.py

```diff
@@ -13,6 +12,18 @@ def __init__(self):
     def get_dict(self):
         return self._dict.copy()
 
+    @staticmethod
```

Comment:
no, if you look at the FileFilter class in Java, there are two methods:
- a *class* method `.or(Collection<FileFilter>)` which creates a new nested "OR" filter from multiple filters in the arguments
- an *instance* method `.or(FileFilter)` which creates a new OR filter from `this` and the argument
I implemented the first one. I can implement the second as well.